pkg/pilot/cassandra/v3/options.go

```diff
@@ -66,6 +72,12 @@ func (o *PilotOptions) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.Master, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	flags.DurationVar(&o.ResyncPeriod, "resync-period", defaultResyncPeriod, "Re-sync period for control loops operated by the pilot")
 	flags.StringVar(&o.ConfigDir, "config-dir", defaultConfigDir, "Base directory for additional Pilot configuration")
+	flags.StringVar(
+		&o.JolokiaURL,
+		"jolokia-url",
+		"",
+		"The base URL of the Jolokia REST API server",
+	)
```

Comment:
Can you collapse this into a single line for consistency with other lines in this fn?